### ISSUE TITLE
Kotlin docs use \$$1 instead of $1 in example SQL

### ISSUE THREAD

**Issue by tobiasmuehl** [eclipse-vertx/vertx-sql-client#443]
Apologies for probably opening this issue in the wrong repo.

https://vertx.io/docs/vertx-pg-client/kotlin/#_handling_arrays

Shows this example query:

```kotlin
client.preparedQuery("SELECT address, (address).city FROM address_book WHERE id=\$$1", Tuple.of(3), { ar ->
....
```

The `\$$1` is not valid Postgres SQL and will throw an Exception. The Java version correctly uses `$1`.

**Comment by tobiasmuehl** [comment]
Well now the docs show "Code not translatable", so that somewhat fixes this..

**Comment by tsegismont** [comment]
Escaping issue tracked by https://github.com/vert-x3/vertx-codetrans/issues/49

**Comment by tsegismont** [comment]
Code translation issue tracked by https://github.com/vert-x3/vertx-codetrans/issues/51

**Comment by tsegismont** [comment]
Fixed in vert-x3/vertx-codetrans#49 and vert-x3/vertx-codetrans#51
